composer.json

```diff
@@ -7,19 +7,19 @@
     "prefer-stable": true,
     "require": {
         "php": ">=8.1",
-        "drupal/cas": "^2.3",
-        "drupal/core": "^10"
+        "drupal/cas": "^3.0",
+        "drupal/core": "^10.3 || ^11.0"
     },
     "require-dev": {
         "composer/installers": "^1.11",
         "drupal/cas_mock_server": "^2.0.1",
-        "drupal/core-composer-scaffold": "^10",
+        "drupal/core-composer-scaffold": "^10.3 || ^11.0",
         "drupal/drupal-extension": "^5.0",
-        "drush/drush": "^12",
+        "drush/drush": "^12 || ^13",
         "mikey179/vfsstream": "^1.6.10",
         "nikic/php-parser": "^4",
         "openeuropa/behat-transformation-context": "^0.2",
-        "openeuropa/code-review": "^2.0.0-alpha6",
+        "openeuropa/code-review-drupal": "^1.0.0-alpha",
         "openeuropa/task-runner-drupal-project-symlink": "^1.0-beta6",
```

Comment:
we still need to sort out the task runner:
- this merged https://github.com/openeuropa/task-runner/pull/173/files
- this sorted out https://github.com/openeuropa/task-runner-drupal-project-symlink/blob/1.x/composer.json#L11 (wasn't pointing to a release)
- and then some releases for these 2 packages.

Reply:
done